js/extjs/resources/css/ytheme-magento.css

```diff
@@ -628,4 +628,4 @@ input.l-tcb {
 }
 .ext-ie .x-tree {
     position:static !important;
-}
\ No newline at end of file
+}
```

Comment:
for some reason, github editor is so stupid, you cannot remove line break from end of file